app/assets/javascripts/admin/addon/components/admin-config-area-card.gjs

```diff
@@ -0,0 +1,16 @@
+import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import i18n from "discourse-common/helpers/i18n";
+
+export default class AdminConfigAreaCard extends Component {
+  @tracked collapsed = false;
```

Comment:
We should remove this for now until we get around to this functionality

Reply:
ESLint complains about `Do not create empty backing classes for Glimmer template tag only components  ember/no-empty-glimmer-component-classes` if I remove the property and I think the only way to stop this error is to remove the component and inline the template in `admin-config-areas/about.gjs` which I don't want to do.

Reply:
ah yeah fair enough since we are expanding this anyway :+1: